src/sonic-yang-models/tests/files/sample_config_db.json

```diff
@@ -1207,9 +1207,9 @@
         "FEATURE": {
             "bgp": {
                 "auto_restart": "enabled",
-                "has_global_scope": "false",
```

Comment:
>false [](http://example.com/codeflow?start=37&length=5)

Is this really needed? I see the boolean_type accepts lowercases.

Reply:
I have added these changes with "False" and "True" with a capital letter to make it clear that this option can also be used and it is valid.

Reply:
The same in the #9418

Reply:
Got it.